src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+:root {
+  --colorBlye: #00acdc;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("/fonts/Roboto-Medium.ttf");
+  font-weight: 500;
+  font-style: normal;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__link {
+  margin: 10px 0;
+}
+
+.nav__list {
+  margin-top: 19px;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+  font-size: 12px;
+  padding-top: 23px;
```

Comment:
you need to add also padding-bottom here. It will help to pass your tests. Don't forget to reset default paddings and margins of ul and li